.travis.yml

```diff
@@ -31,121 +32,121 @@ matrix:
     bundler_args:
     script: bundle exec rake beaker
   - rvm: 2.4.4
     sudo: required
     services: docker
-    env: BEAKER_set="ubuntu-1604" BEAKER_PUPPET_COLLECTION=puppet5
+    env: BEAKER_set="ubuntu-1604" BEAKER_PUPPET_COLLECTION=puppet5 SENSU_ENTERPRISE_FILE=tests/sensu_license.json
     bundler_args:
     script: bundle exec rake beaker
   - rvm: 2.5.3
     sudo: required
     services: docker
-    env: BEAKER_set="ubuntu-1604" BEAKER_PUPPET_COLLECTION=puppet6
+    env: BEAKER_set="ubuntu-1604" BEAKER_PUPPET_COLLECTION=puppet6 SENSU_ENTERPRISE_FILE=tests/sensu_license.json
     bundler_args:
     script: bundle exec rake beaker
   - rvm: 2.4.4
     sudo: required
     services: docker
-    env: BEAKER_set="debian-8" BEAKER_PUPPET_COLLECTION=puppet5
+    env: BEAKER_set="debian-8" BEAKER_PUPPET_COLLECTION=puppet5 SENSU_ENTERPRISE_FILE=tests/sensu_license.json
     bundler_args:
     script: bundle exec rake beaker
   - rvm: 2.5.3
     sudo: required
     services: docker
-    env: BEAKER_set="debian-8" BEAKER_PUPPET_COLLECTION=puppet6
+    env: BEAKER_set="debian-8" BEAKER_PUPPET_COLLECTION=puppet6 SENSU_ENTERPRISE_FILE=tests/sensu_license.json
     bundler_args:
     script: bundle exec rake beaker
   - rvm: 2.4.4
     sudo: required
     services: docker
-    env: BEAKER_set="ubuntu-1404" BEAKER_PUPPET_COLLECTION=puppet5
+    env: BEAKER_set="ubuntu-1404" BEAKER_PUPPET_COLLECTION=puppet5 SENSU_ENTERPRISE_FILE=tests/sensu_license.json
     bundler_args:
     script: bundle exec rake beaker
   - rvm: 2.5.3
     sudo: required
     services: docker
-    env: BEAKER_set="ubuntu-1404" BEAKER_PUPPET_COLLECTION=puppet6
+    env: BEAKER_set="ubuntu-1404" BEAKER_PUPPET_COLLECTION=puppet6 SENSU_ENTERPRISE_FILE=tests/sensu_license.json
     bundler_args:
     script: bundle exec rake beaker
   - rvm: 2.4.4
     sudo: required
     services: docker
-    env: BEAKER_set="ubuntu-1804" BEAKER_PUPPET_COLLECTION=puppet5
+    env: BEAKER_set="ubuntu-1804" BEAKER_PUPPET_COLLECTION=puppet5 SENSU_ENTERPRISE_FILE=tests/sensu_license.json
     bundler_args:
     script: bundle exec rake beaker
   - rvm: 2.5.3
     sudo: required
     services: docker
-    env: BEAKER_set="ubuntu-1804" BEAKER_PUPPET_COLLECTION=puppet6
+    env: BEAKER_set="ubuntu-1804" BEAKER_PUPPET_COLLECTION=puppet6 SENSU_ENTERPRISE_FILE=tests/sensu_license.json
     bundler_args:
     script: bundle exec rake beaker
   - rvm: 2.4.4
     sudo: required
     services: docker
-    env: BEAKER_set="amazonlinux-2" BEAKER_PUPPET_COLLECTION=puppet5
+    env: BEAKER_set="amazonlinux-2" BEAKER_PUPPET_COLLECTION=puppet5 SENSU_ENTERPRISE_FILE=tests/sensu_license.json
     bundler_args:
     script: bundle exec rake beaker
   - rvm: 2.5.3
     sudo: required
     services: docker
-    env: BEAKER_set="amazonlinux-2" BEAKER_PUPPET_COLLECTION=puppet6
+    env: BEAKER_set="amazonlinux-2" BEAKER_PUPPET_COLLECTION=puppet6 SENSU_ENTERPRISE_FILE=tests/sensu_license.json
```

Comment:
in the spec_helper or related place could we just specify that if the environment variable `SENSU_ENTERPRISE_FILE` is set to use it and if not set it to the default `tests/sensu_license.json`. That would DRY up the configuration.

Reply:
Updated with that change.